src/module.js

```diff
@@ -1628,245 +1635,6 @@ export default class DailyIframe extends EventEmitter {
     }
   }
 
-  // fix this later to be a no-op
-  fixupParticipant(msgOrP) {
```

Comment:
this method seemed safe to get rid of—participant properties were already being set properly on the call machine side of things.